packages/flutter/lib/src/material/carousel.dart

```diff
@@ -58,11 +88,42 @@ class CarouselView extends StatefulWidget {
     this.onTap,
     required this.itemExtent,
```

Comment:
Should developers be able to pass `null` in here, or should this be instead:

```suggestion
    required double this.itemExtent,
```